commit b61973e3ca958a67a238b2e32dcbf9c99e57bb93
Author: Boris Brezillon <boris.brezillon@free-electrons.com>
Date:   Thu Feb 2 11:37:36 2017 +0100

    ARM: dts: bcm283x: Add HDMI audio related properties
    
    Add the dmas and dma-names properties to support HDMI audio.
    
    Signed-off-by: Boris Brezillon <boris.brezillon@free-electrons.com>
    Signed-off-by: Eric Anholt <eric@anholt.net>
    (cherry picked from commit d46d2c6380c10e80e99f6af9067356128bffac6b)

diff --git a/arch/arm/boot/dts/bcm283x.dtsi b/arch/arm/boot/dts/bcm283x.dtsi
index 348c71bbc913..7d58cd79ac6b 100644
--- a/arch/arm/boot/dts/bcm283x.dtsi
+++ b/arch/arm/boot/dts/bcm283x.dtsi
@@ -339,6 +339,8 @@
 			clocks = <&clocks BCM2835_PLLH_PIX>,
 				 <&clocks BCM2835_CLOCK_HSM>;
 			clock-names = "pixel", "hdmi";
+			dmas = <&dma 17>;
+			dma-names = "audio-rx";
 			status = "disabled";
 		};
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1480_b61973e3ca958a67a238b2e32dcbf9c99e57bb93.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1480_b61973e3ca958a67a238b2e32dcbf9c99e57bb93.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
